server.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+require('./data/weather.json')
+require('dotenv').config();
+const express =require('express');
+const cors = require('cors');
+const list = require('./data/weather.json');
+
+const app = express();
+
+app.use(cors());
+
+const PORT = process.env.PORT;
+
+
+
+
+
+app.get('/weather', handleGetWeather)
+
+
+
+
+
```

Comment:
Good work on the lab so far!

Generally one line of space can separate routes vs modules. We can clean up this extra whitespace for easier readability and best practices.

- we can move line 8 to line 7
- use one space to separate out routes, modules, and middleware 👍